### PR TITLE
feat: [CCM-7889]: Azure Filter Support in Anomalies Filter Panel

### PR DESCRIPTION
**Pull request by rajarshee-harness**
##### Jira Links:

https://harness.atlassian.net/browse/CCM-7889

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by rajarshee-harness** [comment]
retrigger titlecheck

File: src/modules/75-ce/components/AnomaliesFilter/FilterDrawer/utils.ts [hunk 1]
```diff
@@ -116,6 +131,15 @@ export const getAnomalyFilterPropertiesFromForm = (formData: AnomaliesFilterForm
   if (k8sWorkloadNames) {
     filterProperties.k8sWorkloadNames = getValueFromOption(k8sWorkloadNames)
   }
+  if (azureResourceGroups) {
```
Comment (rishik30) [inline src/modules/75-ce/components/AnomaliesFilter/FilterDrawer/utils.ts:134]: create a utility here that takes in formData and based on the keys generates filterProperties.
```js
const getPropertiesFromFormData = (formData) => {
  const filterProperties = {}
  Object.entries(formData).forEach(([key, value]) => {
    filterProperties[key] = getValueFromOption(value)
  })
  return filterProperties
}
```

**Review by rishik30 — CHANGES_REQUESTED** [review]

File: src/modules/75-ce/components/AnomaliesFilter/FilterDrawer/utils.ts (hunk 1 above)
Comment (rajarshee-harness) [reply src/modules/75-ce/components/AnomaliesFilter/FilterDrawer/utils.ts]: Done

**Comment by rajarshee-harness** [comment]
trigger cypress

**Review by rishik30 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**